oeps/processes/oep-0001.rst

```diff
@@ -47,6 +47,20 @@ OEP-1: OEP Purpose and Guidelines
   :local:
   :depth: 1
 
+TL;DR
+=====
+
+* An Open edX Proposal specifies a best practice, architectural decision, or
+  community process that the Open edX communty has agreed should be adopted by
+  the project.
+* OEPs are written about broad architectural changes and community-wide
+  processes and practices. They are not used to dictate small decisions made in
+  every day feature work.
```

Comment:
Although I agree with this, the OEP doesn't actually say it, so I don't think it should be in the TL;DR.

Reply:
I was trying to summarize the ideas in the following sentences:

>  An OEP is a document that details a specific technology decision being made by the Open edX community, in the form of a best practice, architecture design, or process adjustment. An OEP should provide the use cases and rationales that surround that choice. 

> Small enhancements or patches often don’t need an OEP and can be injected into the Open edX development workflow with a patch submission.

I may have injected too much of the how-it-currently-works into that statement. Is there a better way of quickly indicating something like, "OEPs are big and there are easier ways to effect smaller changes"?

Reply:
> Small enhancements or patches often don’t need an OEP and can be injected into the Open edX development workflow with a patch submission.

I read this as a funny way of saying that "you don't need an OEP to accompany every single PR." Which, on second thought, does sort of capture what your TL;DR bullet is saying. Aside: it would be nice if the OEP also said "for medium-size, repo-specific decisions you can write an ADR instead".

Anyway, I think the bullet point is good to leave in, although to really lean into short-and-snappiness I recommend phrasing it as simply:
> * OEPs are *not* used to dictate small decisions made in every day feature work.

Reply:
I like that @kdmccormick . What do you think about:

```
OEPs are not used to dictate small decisions made in every day feature work. See [OEP--19](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html?highlight=documentation#agile-documentation-types) for more detail.
```

Reply:
👍🏻